lms/djangoapps/discussion/templates/discussion/discussion_board_fragment.html

```diff
@@ -44,23 +44,36 @@ <h3>${course_organization}</h3>
         <header class="page-header has-secondary">
             ## Breadcrumb navigation
             <div class="page-header-main">
-                <nav aria-label="${_('Discussions')}" class="sr-is-focusable" tabindex="-1">
+                <nav aria-label="${_('Discussions')}" class="sr-is-focusable " tabindex="-1">
                     <div class="has-breadcrumbs"></div>
-                    <div class="select-box">
-                      <select class="cz-custom-select">
-                        <option value="">All Topics</option>
-                      </select>
+
+                    <div class="dropdown" style="float: left">
+                        <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">All Topics
+                            <span class="caret"></span></button>
+                        <ul class="dropdown-menu" style="width: 300px;">
+                            <input class="form-control" id="all-topics-filter" type="text" placeholder="Filter..">
+                            <span id="all-topics-main">
+
+                            </span>
+                        </ul>
                     </div>
-                    <div class="select-box">
-                      <select class="cz-custom-select">
-                        <option value="">All Discussions</option>
-                      </select>
+
+                    <div class="dropdown" style="float: left; padding-left: 15px;">
+                        <button class="btn dropdown-toggle" type="button" data-toggle="dropdown">All Discussions
+                            <span class="caret"></span>
+                        </button>
+                        <ul class="dropdown-menu" id="all-discussions-main" style="width: 300px;">
+                        </ul>
                     </div>
-                    <div class="select-box">
-                      <select class="cz-custom-select">
-                        <option value="">Posts I am Following</option>
-                      </select>
+
+                    <div class="dropdown" style="float: left; padding-left: 15px;">
```

Comment:
can we move styling in css files?

Reply:
Actually its just temporary and Rahat bhai said he will work on the styling later according to colaraz standard. so I put it here which makes it easy to understand.